.github/workflows/image-push.yaml

```diff
@@ -49,6 +55,8 @@ jobs:
           push: true
           tags: ghcr.io/${{ env.REPOSITORY_LC }}:${{  github.ref_name }}
           file: Dockerfile
+          build-args: |
```

Comment:
Not related to this change, but now I thing about it, I find it a bit weird we're not using here `make build; make push`.

Reply:
We have it git actions style, it is nicer

it is bit less good that not both sides are doing the exactly same flow
but this is the drill not just here

Reply:
For example in this PR you have to update two places with the same logic. I find it bug prune.

Reply:
generally of course it is better to have the same flow, but i dont think it deserve neglecting the git actions plugins

we can remove the update of the `latest` one since we dont care as it always change
this will reduce the complexity, if you prefer

Note that this file doesnt change almost, so it also worth to consider it
maybe there is a way with git actions the have global build-args in few sections and then it will be used one
(but i dont know to do it by heart)

Reply:
Let's at least change the Makefile to push to the local repo and not to some quay.
Can be done in a separate PR.

Reply:
https://github.com/AlonaKaplan/kubesecondarydns/pull/30/files